tests/core/pyspec/eth2spec/test/helpers/execution_payload.py

```diff
@@ -1,4 +1,8 @@
-def build_empty_execution_payload(spec, state):
+def build_empty_execution_payload_with_randao(spec, state):
+    return build_empty_execution_payload(spec, state, spec.Bytes32())
```

Comment:
This is a stubbed randao, rather than the real thing, no?

The name of this function implies to me that it will have a correct randao reveal